docs/build/references/command-line-interface.md

```diff
@@ -228,11 +248,19 @@ It means that this node will sample all nodes, not just validators.
 
 `--staking-tls-cert-file` (string, file path):
 
-Avalanche uses two-way authenticated TLS connections to securely connect nodes. This argument specifies the location of the TLS certificate used by the node. By default, the node expects the TLS certificate to be at `$HOME/.avalanchego/staking/staker.crt`.
+Avalanche uses two-way authenticated TLS connections to securely connect nodes. This argument specifies the location of the TLS certificate used by the node. By default, the node expects the TLS certificate to be at `$HOME/.avalanchego/staking/staker.crt`. This flag is ignored if `--staking-tls-cert-file-content` is specified.
+
+`--staking-tls-cert-file-content` (string):
+
+As an alternative to `--staking-tls-cert-file`, it allows specifying base64 encoded of the TLS certificate used by the node.
```

Comment:
A possible missing word here `...specifying base64 encoded contents?/file? of the TLS...`

Reply:
Done